mitm_service/tunneler/tunneler.py

```diff
@@ -155,7 +155,11 @@ def forward_loop(self):
             if recv_ipheader.dst_ip != self.target_ip and recv_ipheader.src_ip != self.target_ip:
                 continue
 
-            disrupted_packet = self.disrupt_layers(data)
+            try:
+                disrupted_packet = self.disrupt_layers(data)
+            except DropPacketException:
+                # some filter in disrupt_layers raised a drop packet exception, so drop this packet
+                continue
```

Comment:
write log ?

Reply:
No need, if this exception is raised it means a user-supplied filter raised it on purpose, so they can print a log if they want.